WeatherUI/Views/CityWeatherView/CityWeatherView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CityWeatherView.swift
+//  WeatherUI
+//
+//  Created by kostya on 04.07.2022.
+//
+
+import SwiftUI
+
+struct CityWeatherView: View {
+    let model: CityWeather
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0.0) {
+                HStack {
+                    Text(model.cityName)
+                        .font(.system(size: 18, weight: .regular))
+                    Image("city_pin")
+                }
+                HStack(alignment: .center) {
```

Comment:
thats a good point about swiftui style guide. i didn't find any, although we can have some essential agreements towards this.
speaking of this particular point - it is arguable. basically most of entities of swiftui are views, but i believe we shouldn't separate each with empty line. we could separate some grouping views, like lists, grids, etc. but i'm not sure this gives a lot more readability since grouping views are already noticeable due to indentation.